.releaserc.json

```diff
@@ -0,0 +1,3 @@
+{
+	"branches": ["semantic-release"]
```

Comment:
ultimately, the release branch should be `dev` as this is the (default) branch the PR are merged into